tests/odm/conftest.py

```diff
@@ -197,118 +297,38 @@ async def session(cli):
     await s.end_session()
 
 
+@pytest.fixture()
+async def deprecated_init_beanie(db):
+    for model in TESTING_MODELS:  # crude clear from init
+        await model.get_motor_collection().drop()
+        await model.get_motor_collection().drop_indexes()
+    with warnings.catch_warnings(record=True) as w:
+        warnings.simplefilter("always")
+
+        await init_beanie(
```

Comment:
Only one slight "issue" is that now `init_beanie` will be executed twice (but I guess this is fine), since there is an autouse `init` fixture in this file as well.

Reply:
yes geting rid of auto use would be hard.... i am fully aware of it...